setup.py

```diff
@@ -1,10 +1,17 @@
 import json
 import os
+import sys
 
 import setuptools
 
 loc = os.path.dirname(os.path.abspath(__file__))
 
+micro = False
+if '--micro' in sys.argv:
+    micro = True
+    sys.argv.remove('--micro')
```

Comment:
you don't need to remove it from the args

Reply:
![image](https://user-images.githubusercontent.com/4350002/107532166-9935d400-6b8b-11eb-8ef1-39686bb24642.png)

I could not get it to work without removing it. I think the extra arg was getting passed into setup. Althought to be fair I also probably make the wheels differently than you guys. So maybe thats why.